release-1.5/release-notes-draft.md

```diff
@@ -128,6 +128,16 @@ release [38537](https://github.com/kubernetes/kubernetes/issues/38537)
 
 Continuous integration builds have used the following versions of external dependencies, however, this is not a strong recommendation and users should consult an appropriate installation or upgrade guide before deciding what versions of etcd, docker or rkt to use. 
 
-* Docker version 1.11.2, Docker version 1.12.0 [has also been validated](https://github.com/kubernetes/kubernetes/issues/28698)
+* Docker version 1.11.2
```

Comment:
Or we can re-use what we had for 1.4 release:

If you are using Kubernetes to manage docker containers, please be aware Kubernetes has been validated to work with docker 1.9.1, docker 1.11.2 (#23397), and docker 1.12.0 (#28698) here following with issues for each release.